ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  test-image:
+    name: strimzi-tests
+    namespace: ci
+    tag: release-0.32.x
+build_root:
```

Comment:
You can also remove build_root since you mirrored your image as a base_image. We'd only need this build_root image if we were using it to build other images on the fly.